### PR TITLE
Add in basic Images/ Collections models

### PR DESCRIPTION
**Pull request by mejackreed**
This PR adds in the basic image/collection models.

Sorry about all of the hound chatter. These comments should all be resolved by #24 .

TLDR: Hound is commenting on automatically created files which we should not edit.

### CODE REVIEW

**Review by versae — APPROVED** [review]

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+image_files = Rails.root.join(Settings.IMAGE_PATH, '*.jpg')
```
Comment (aeschylus) [inline db/seeds.rb:8]: Are the images jpegs, not JPEG2000?

Comment (mejackreed) [reply db/seeds.rb:8]: Images will most likely be tifs... this is just for a convenience at the moment.

Comment (versae) [reply db/seeds.rb:8]: The ones I have are actually TIFF

Comment (mejackreed) [reply db/seeds.rb:8]: Ok cool. Created a ticket here for this: https://github.com/sul-cidr/histonets/issues/27

**Pull request merged**